databroker/src/broker.rs

```diff
@@ -679,6 +719,28 @@ impl Subscriptions {
                 }
             }
         });
+
+        self.actuation_subscriptions.retain(|sub| {
+            if !sub.actuation_provider.is_available() {
+                info!("actuation provider gone: removing subscription");
+                false
+            } else {
+                match sub.permissions.expired() {
+                    Ok(()) => true,
+                    Err(PermissionError::Expired) => {
+                        info!("Token expired for actuation provider: removing subscription");
+                        false
+                    }
+                    Err(err) => {
+                        info!(
```

Comment:
This is not expected, right? As expired() only can return Ok and Expired right?

Change to `warn!` and included "unexpected" in text or comment?

Reply:
Changed to a more generic: 
```
match &sub.permissions.expired() {
    Ok(()) => true,
    Err(permission_error) => {
        info!("Removing Provider due to {:?}", permission_error);
        false
    }
}
```